IntroToCSharp/Pages/Activities/Booleans.razor

```diff
@@ -0,0 +1,111 @@
+    </ChildContent>
+</Questions>
+
+<Section Title="Exploring Booleans">
+<p>
+    	We can see from this that booleans can be a lot of fun to play with by themselves but where they really shine is when you start combining 
+        them with other pieces of code! Booleans thrive when working in together with other pieces of code such as <Keyword Word="conditional operators"/> and <Keyword Word="If-Else Statements"/>, 
+        along with a whole lot of other tools at our disposal.
+</p>
+</Section>
+
+<Questions Title="Testing your knowledge!">
+    <ChildContent>
+        <p>Are these Questions Booleans?</p>
+        <p>Replace me with multiple choice quiz</p>
+        <Question Text="Is it going to rain today?" />
+        <Question Text="1 + 1 = 2." />
+        <Question Text="Your shirt is red." />
+        <p> Answers; yes, no, no.</p>
+    </ChildContent>
+</Questions>
+
+<Section Title="Combining Colors!">
+    <p>
+        Let’s take a look at another problem here! 
+    </p>
+    <p>
+
+        For this code we are taking two colors and checking to see if they make purple. 
+        In this case we are using an IF Statement with a boolean to see what the result of the question "Is ColorOne red & Is ColorTwo blue?"" If <Keyword Word ="both"/> of those statements are true, 
+        then we are going to print that the color is purple! If those statements aren’t true then we are going to print that those colors aren't purple.
+    </p>
+
+    <CodeBlock Filename="Booleans/CombiningColors.cs" />
+
+</Section>
+
+<Section Title="How Computers Understand Booleans!">
+    <p>
+        Something know about booleans is that they don’t actually return true or false. 
+        What booleans are really returning is a 1 for true or a 0 for false.
+    </p>
+    <p>
+        You might have noticed that sometimes when you exit a program your terminal might say something along the lines of return with a value of 0. 
+        This is actually answering the question “Is there a problem running this program?” A value of 0 means the program ran perfectly fine according to the computer 
+        (although not always according to us humans!) While a 1 is usually accompanied with some type of code saying what went wrong with the program!
+    </p>
+</Section>
+
+<Section Title="Your Turn!">
+
+    <p>
+        Over the course of this lesson we looked at booleans and how they can be used to determine if something is true or false.
+        Now it is your turn to try things out!
+    </p>
+    <p>
+        For this preparation we are going to take the previous challenge and expand it to a few different colors!
+        This time however when the program runs you should get the color that ColorOne & ColorTwo combine to.
+    </p>
+    <p>
+        For example, if I ColorOne was "red" and ColorTwo was "yellow" you would get "orange". You are going to do this for every color in the rainbow!
```

Comment:
where do they write the code for this challenge?

Reply:
In their own project.